packages/api/src/controllers/reviews/v1/mod.rs

```diff
@@ -88,22 +97,47 @@ impl ReviewControllerV1 {
     pub async fn update(&self, id: i64, params: ReviewUpdateRequest) -> Result<Json<Review>> {
         tracing::debug!("update review: {:?} {:?}", id, params);
 
-        Ok(Json(Review::default()))
+        let fetched_review = sqlx::query_as!(
+            Review,
+            "UPDATE reviews SET name = $1, image = $2, review = $3 WHERE id = $4 RETURNING *",
+            params.name,
+            params.image,
+            params.review,
+            id
+        )
+        .fetch_one(&self.pool)
+        .await;
+
+        match fetched_review {
+            Ok(review) => Ok(Json(review)),
+            Err(_) => Err(ApiError::ApiCallError("Something went wrong".to_string())),
+        }
```

Comment:
It's good.
But some case, you may simplize this code by ReviewRepository's `update` function.
- Refer to https://github.com/biyard/rust-sdk/blob/main/packages/by-macros/tests/update_into_test.rs#L81